src/test/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImplTest.java

```diff
@@ -239,4 +274,78 @@ private MediaRequestEntity createMediaRequest(OffsetDateTime startTime, OffsetDa
+
+        audioTransformationService.notifyUser(mockMediaRequestEntity, mockCourtCase, NotificationApi.NotificationTemplate.ERROR_PROCESSING_AUDIO.toString());
+
+        verify(mockNotificationApi).scheduleNotification(dbNotificationRequestCaptor.capture());
+        var actual = dbNotificationRequestCaptor.getValue();
+        assertEquals(actual.getTemplateValues().get(DEFENDANTS), NO_DEFENDANTS);
+
+    }
+
+    @SneakyThrows
+    @Test
+    void testNotifyUserScheduleErrorNotificationWithDefendants() {
+        List<DefendantEntity> defendantList = new ArrayList<>();
+        defendantList.add(mockDefendantEntity);
+
+        initNotifyUserScheduleErrorNotificationMocks();
+        when(mockDefendantEntity.getName()).thenReturn(TEST_DEFENDANT_NAME);
+        when(mockCourtCase.getDefendantList()).thenReturn(defendantList);
+
+        audioTransformationService.notifyUser(mockMediaRequestEntity, mockCourtCase, NotificationApi.NotificationTemplate.ERROR_PROCESSING_AUDIO.toString());
+
+        verify(mockNotificationApi).scheduleNotification(dbNotificationRequestCaptor.capture());
+        var actual = dbNotificationRequestCaptor.getValue();
+        assertEquals(actual.getTemplateValues().get(DEFENDANTS), TEST_DEFENDANT_NAME);
+
+    }
+
+    @SneakyThrows
+    @Test
+    void testNotifyUserScheduleErrorNotificationMissingHearingDataAndCourthouseName() {
+        List<DefendantEntity> defendantList = new ArrayList<>();
+        defendantList.add(mockDefendantEntity);
+
+        when(mockMediaRequestEntity.getId()).thenReturn(1);
+        when(mockHearing.getHearingDate()).thenReturn(null);
+        when(mockMediaRequestEntity.getHearing()).thenReturn(mockHearing);
+        when(mockMediaRequestEntity.getStartTime()).thenReturn(TIME_12_00);
+        when(mockMediaRequestEntity.getEndTime()).thenReturn(TIME_13_00);
+        when(mockMediaRequestEntity.getRequestor()).thenReturn(mockUserAccountEntity);
+        when(mockUserAccountEntity.getId()).thenReturn(1);
+        when(mockHearing.getCourtCase()).thenReturn(mockCourtCase);
+        when(mockCourtCase.getCourthouse()).thenReturn(mockCourthouse);
+        when(mockCourthouse.getCourthouseName()).thenReturn(null);
+        when(mockUserAccountRepository.findById(any())).thenReturn(Optional.of(mockUserAccountEntity));
+        when(mockDefendantEntity.getName()).thenReturn(TEST_DEFENDANT_NAME);
+        when(mockCourtCase.getDefendantList()).thenReturn(defendantList);
+
+        audioTransformationService.notifyUser(mockMediaRequestEntity, mockCourtCase, NotificationApi.NotificationTemplate.ERROR_PROCESSING_AUDIO.toString());
+
+        verify(mockNotificationApi).scheduleNotification(dbNotificationRequestCaptor.capture());
+        var actual = dbNotificationRequestCaptor.getValue();
+        assertEquals(actual.getTemplateValues().get(HEARING_DATE), TEST_NA);
+        assertEquals(actual.getTemplateValues().get(HEARING_DATE), TEST_NA);
```

Comment:
Why are you testing 'HEARING_DATE' twice? 

Also, think you're missing a few scenarios:
- courthouse name is blank or empty
- can audio start/end time ever be empty? Looks like code wouldn't be able to handle this scenario so might be worth putting some defensive code in to check?

Reply:
Duplication removed and tests refactored to provide clarity of the following:

- test with no defendants
- test with defendant
- test with no courthouse name
- test with no hearing date
- test with no start time
- test with no end time

Confirmation of missing default string for both start and end times from Leanne Jacobs to be set as "N/A"